### ISSUE TITLE
attention_mask_label

### ISSUE THREAD

**Issue by gui-li** [kamalkraj/BERT-NER#34]
I have noticed attention_mask_label in your experiment code, but why you set it to none to avoid using it? Is the performance worse if you only use active parts of loss?

**Comment by tcqiuyu** [comment]
Same question here

**Comment by kamalkraj** [comment]
@gui-li @tcqiuyu 
Using attention mask label and `ignore_index = 0` has similar performance.
